### PR TITLE
Modal component

### PR DESCRIPTION
**Pull request by konsumer**
This adds a modal component & docs, that looks like this:

screen shot 2018-10-30 at 12 42 20 pm

### CODE REVIEW

**Comment by konsumer** [comment]
I tried to follow what notifications is doing, but also leave it up to the user what to put inside modal (in this example, I use a card to make a nice material-looking modal.)

File: src/Modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    show = content => {
+        this.setState({ content });
+    };
+
+    hide = () => {
+        this.setState({ content: false });
+    };
+
+    render() {
+        const { style = {}, className } = this.props;
```
Comment (sabeurthabti) [inline src/Modal/Modal.js]: you get props as part of the render

Comment (sabeurthabti) [reply src/Modal/Modal.js]:
```
render({ style = {}, className }) {


```

**Comment by sabeurthabti** [comment]
Really cool - I will play around with this tonight/tomorrow.

File: src/Modal/Modal.js (hunk 1 above)
Comment (konsumer) [reply src/Modal/Modal.js]: That makes sense, and I am happy to make the change, but I tried to closely follow the [style of `Notify`](https://github.com/ajainvivek/preact-fluid/blob/master/src/Notify/Notify.js#L172). Should I add to the PR?

Comment (sabeurthabti) [reply src/Modal/Modal.js]: no its fine.

**Comment by sabeurthabti** [comment]
@konsumer Thank you for the contribution :)

**Comment by sabeurthabti** [comment]
@konsumer has been released as part of `v0.9.0`

**Pull request merged**